### PR TITLE
[Issue #1] Use the game folder instead of the stories folder

### LINKED ISSUE #1: [Feature request] Only ask for path to game folder, not stories
**Is your feature request related to a problem? Please describe.**
Many players won't know where the Stories file is. Asking them for their game folder (example: "C:\Program Files (x86)\Steam\steamapps\common\Exocolonist") is probably a better experience. It's also what they get if they go to Properties > Local Files > Browse in Steam.

**Describe the solution you'd like**
Instead of asking the user for the path to their game's Stories folder, ask for (and store) the path to their Exocolonist game folder.
Append Exocolonist_Data/StreamingAssets/Stories as necessary to paths.

**Describe alternatives you've considered**
¯\\_(ツ)_/¯ Not doing this because maybe no one will care?

**Additional context**
This requires a schema update for user settings. This is a great time to establish a pattern for the versioning system, including backwards-comparability / auto-upgrades.

### PR DESCRIPTION
**Pull request by eskolnik**
# Description

- In setup, prompt user for game folder instead of stories folder
- Supply a default value if the user enters an empty string
Fixes #1 

## Type of Change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Run locally, and successfully loaded a mod after each test:
- Without a settings file, and provided a path. Settings were persisted successfully.
- Without a settings file, and provided an empty string. The new default path setting was applied successfully.
- With an existing settings file. The script skipped the prompt for a path as expected before this patch.

# Checklist:

- [x] I have performed a self-review of my own diff
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have included corresponding changes to the documentation

# Screenshots

Please include screenshots if applicable, especially when user interfaces have been updated.

### CODE REVIEW

File: src/loader.ps1 [hunk 1]
```diff
@@ -208,7 +210,17 @@ function getSettings() {
     if (!$isSettingsFromFile) {
         $newSettings = [ExomodLoaderSettingsFile]::new()
         $newSettings.SettingsVersion = 1;
-        $newSettings.PathToGameStoriesFolder = Read-Host -Prompt "Please enter the path of your game's unmodified Stories directory. Example: $(${ENV:ProgramFiles(x86)})\Steam\steamapps\common\Exocolonist\Exocolonist_Data\StreamingAssets\Stories"
+
+        # Prompt the user for the path to their game folder. Use the default path if none is provided.
+        $userGameDirectory = Read-Host -Prompt "Please enter the path of your game directory. Press enter to use the default value: $($defaultPathToGameFolder)"
```
Comment (suzicurran) [inline src/loader.ps1:215]: "Please enter [...] Press enter..."
Can we vary this a bit for readability? Examples: "Please provide..." or "Press the ENTER key..."

File: src/loader.ps1 [hunk 2]
```diff
@@ -208,7 +210,17 @@ function getSettings() {
...
+
+        # Append the Stories folder to the base game path
+        if ($userGameDirectory -eq "") {
```
Comment (suzicurran) [inline src/loader.ps1:218]: Nice addition!

Comment (eskolnik) [reply src/loader.ps1:218]: Thanks :)

File: src/loader.ps1 (hunk 1 above)
Comment (eskolnik) [reply src/loader.ps1:215]: How about
```suggestion
        $userGameDirectory = Read-Host -Prompt "Please provide the path of your game directory. Leave blank to use the default value: $($defaultPathToGameFolder)"
```

**Review by suzicurran — APPROVED** [review]
LGTM. Thanks for taking on this issue!

**Pull request merged**